frontend/src/ui/wallet/tx/tx-memo-view.js

```diff
@@ -70,7 +65,7 @@ function TxMemoView({transfer, allowMuxed = true}) {
             <input type="text" value={value} onChange={setMemoValue} placeholder={getPlaceholder(type)} ref={useAutoFocusRef}/>
         </div>}
         {!!allowMuxed && type === 'id' && <label className="micro-space text-tiny">
-            <input type="checkbox" checked={transfer.encodeMuxedAddress} onChange={() => transfer.toggleMuxed()}/>{' '}
+            <input type="checkbox" defaultChecked={!!transfer.encodeMuxedAddress} onChange={() => transfer.toggleMuxed()}/>{' '}
```

Comment:
Why did you convert controlled component to uncontrolled?

Reply:
This action, i tried to fix bug: TRANSFER -> set memo = id -> try to select checkbox "Encode as multiplexed address":
with checked={... - doesn't work